source/connections-survive-step-down/tests/README.rst

```diff
@@ -120,9 +120,9 @@ This test requires a replica set with server version 4.0.
 - Execute an insert into the test collection of a ``{test: 1}``
   document.
 - Verify that the insert failed with an operation failure with 10107 code.
-- Verify that the pool has been cleared following the instructions in section `How to verify the connection pool has been cleared`_
 - Execute an insert into the test collection of a ``{test: 1}``
   document and verify that it succeeds.
+- Verify that the pool has been cleared following the instructions in section `How to verify the connection pool has been cleared`_
```

Comment:
Ideally, when using CMAP I would expect the assertion to happen directly after the failed operation. What do you think about changing the position of the assertion based on if it was CMAP/serverStatus?:
```
- Verify that the insert failed with an operation failure with 10107 code.
- If using CMAP, verify that the pool has been cleared (using PoolClearedEvent). 
- Execute an insert into the test collection of a ``{test: 1}``
  document and verify that it succeeds.
- If not using CMAP, verify that the pool has been cleared using serverStatus.
```

Reply:
cleaned the instructions up a bit, and added a bunch of links for clarity, PTAL. Looking forward to converting these tests to "SDAM integration" tests!